templates/cloudflare-workers/remix.config.js

```diff
@@ -1,10 +1,12 @@
 /** @type {import('@remix-run/dev').AppConfig} */
-module.exports = {
-  devServerBroadcastDelay: 1000,
+export default {
   ignoredRouteFiles: ["**/.*"],
   server: "./server.ts",
   serverConditions: ["worker"],
```

Comment:
[Same comment I left on the last one](https://github.com/remix-run/remix/pull/6229#discussion_r1181993844)—IMHO this should be `["workerd", "worker", "browser"]`